### ISSUE TITLE
activity field missing?

### ISSUE THREAD

**Issue by GetTheData** [Fusion-Lifestyle/opendata#16]
I would expect a SessionSeries event to include an activity as [the spec suggests this is a required field](https://www.openactive.io/modelling-opportunity-data/#describing-events-code-schema-event-code-).

However my data profile suggests that activity is only present in around 25% of items:

https://www.getthedata.com/openactive/fusion-lifestyle/data-profile#field_data%2FsuperEvent%2Factivity

The same is roughly true of the superEvent field, under which activity appears to reside:

https://www.getthedata.com/openactive/fusion-lifestyle/data-profile#field_data%2FsuperEvent

Have I interpreted this correctly, i.e. I should expect to see activity for all items, but currently it is only present for around 25%?

**Comment by domfennell** [comment]
Thanks for raising this issue, @GetTheData. We have also noticed a lack of activities throughout much of the Fusion data.

Thanos (@FusionLIfestyle), is this something you could take a look at/advise on, please?

**Comment by FusionLifestyle** [comment]
We had discussed some changes with Nick as far as I remember so the Activity to rely on your OpenActiveManager.

I am not sure if Nick has touch this or it is still in his to-do list.



Many thanks,

Thanos Lempesis
Applications & Development Manager


[cid:image001.jpg@01CDD13F.CDC1D6F0]<http://www.fusion-lifestyle.com/>
Unit 4, Bickels Yard, 151/153 Bermondsey Street, London, SE1 3HA.
T:   020 7740 7524
M: 07825 663294
www.fusion-lifestyle.com<https://217.151.104.114/exchweb/bin/redir.asp?URL=https://217.151.104.114/exchweb/bin/redir.asp?URL=http://www.fusion-lifestyle.com/>
P Think before you print to save energy and paper.

**Comment by nickevansuk** [comment]
That's right, there's a mapping of EventSeries (which are linked to activities) that exists in Fusion's CMS that's not yet fully integrated into this data.

The next step is for Gladstone to install the new version of OWS, after which we can complete the CMS integration work.

**Comment by GetTheData** [comment]
Thanks for clarifying Thanos (@FusionLIfestyle) and @nickevansuk.

Is there anything @domfennell and I could use to map a field currently present in the datafeed onto an activity in the meantime?

**Comment by FusionLifestyle** [comment]
Hi there,

As Nick mentioned there are certain steps to be carried out for better mapping the activity type. There is a plan and there are no other alternatives at the moment or temporary solutions.





That's right, there's a mapping of EventSeries (which are linked to activities) that exists in Fusion's CMS that's not yet fully integrated into this data.

The next step is for Gladstone to install the new version of OWS, after which we can complete the CMS integration work.



Many thanks,

Thanos Lempesis
Applications & Development Manager


[cid:image001.jpg@01CDD13F.CDC1D6F0]<http://www.fusion-lifestyle.com/>
Unit 4, Bickels Yard, 151/153 Bermondsey Street, London, SE1 3HA.
T:   020 7740 7524
M: 07825 663294
www.fusion-lifestyle.com<https://217.151.104.114/exchweb/bin/redir.asp?URL=https://217.151.104.114/exchweb/bin/redir.asp?URL=http://www.fusion-lifestyle.com/>
P Think before you print to save energy and paper.

**Comment by GetTheData** [comment]
Understood - thanks Thanos (@FusionLIfestyle)!

Any idea on a rough timeframe for getting activities into all items?

cc @domfennell @nickevansuk

**Comment by GetTheData** [comment]
Hi Thanos (@FusionLIfestyle) and @nickevansuk,

Just a nudge on this to see if I can get an ETA on full activity data please?

cc @domfennell 

Thanks
Dan
